lisa/tools/stress_ng.py

```diff
@@ -66,10 +66,12 @@ def launch_class_async(
         class_name: str,
         num_workers: int = 0,
         timeout_secs: int = 60,
+        verbose: bool = True,
         sudo: bool = False,
     ) -> Process:
+        v_flag = "-v" if verbose else ""
         return self.run_async(
-            f"--sequential {num_workers} --class {class_name} --timeout {timeout_secs}",
+            f"{v_flag} --sequential {num_workers} --class {class_name} --timeout {timeout_secs}",  # noqa: E501
```

Comment:
This case doesn't need E501, split it to two strings like
```python
f"string1"
f"string2"
```

Reply:
Done.